Assets/Scripts/VrSdk.cs

```diff
@@ -561,9 +560,15 @@ public void SetControllerStyle(ControllerStyle style)
                     break;
                 case ControllerStyle.Neo3:
                     controlsPrefab = m_UnityXRNeo3ControlsPrefab;
+#if PICO_SUPPORTED
```

Comment:
Where are you going with this design? Will Pico be pure OpenXR, or still a distinct build, but based on the same Android XR core (like how Q1 is a different build, but basically Android OpenXR)

Reply:
I want to do this in two stages, we'll give this a shot by adding to OpenXR, but I'm not sure if there's a path forward for passthrough here yet. If it works out, we can remove the current Pico release and just go full OpenXR.

Reply:
ok, makes sense